vendor/calcite-1_25_0/build.gradle

```diff
@@ -18,29 +18,33 @@
 
 plugins { id 'org.apache.beam.vendor-java' }
 
-description = "Apache Beam :: Vendored Dependencies :: Calcite 1.20.0"
+description = "Apache Beam :: Vendored Dependencies :: Calcite 1.25.0"
 
 group = "org.apache.beam"
 version = "0.2"
 
-def calcite_version = "1.20.0"
+def calcite_version = "1.25.0"
 def avatica_version = "1.16.0"
-def prefix = "org.apache.beam.vendor.calcite.v1_20_0"
+def prefix = "org.apache.beam.vendor.calcite.v1_25_0"
 
 List<String> packagesToRelocate = [
         "com.esri",
+        "com.fasterxml",
         "com.google.common",
-        "com.google.thirdparty",
         "com.google.protobuf",
-        "com.fasterxml",
+        "com.google.thirdparty",
         "com.jayway",
         "com.yahoo",
+        "javax.annotation",
+        "net.minidev",
         "org.apache.calcite",
         "org.apache.commons",
         "org.apache.http",
+        "org.apiguardian.api",
         "org.codehaus",
+        "org.objectweb",
         "org.pentaho",
-        "org.yaml"
+        "org.yaml",
 ]
 
 vendorJava(
```

Comment:
some logging libraries need to be put into runtimeDepencies. for you reference: https://github.com/vectorijk/beam/blob/calcite-1-25/vendor/calcite-1_25_0/build.gradle

Reply:
I have copied (+ updated version) of the runtime dependencies part of this.
As this is uncharted territory for me I'm unsure if this is correct now.